### ISSUE TITLE
get_storage_group_snapshot_list fails if no snapshot exists

### ISSUE THREAD

**Issue by mukeshguptakc** [dell/PyU4V#114]
Hello,

While using the Unisphere PyU4V 9.2 observed following exception:

The `conn.replication.get_storage_group_snapshot_list` raises an error if the storage group doesn't has any snapshots.
```
>>> conn.common.get_uni_version()
('V9.2.0.6', '92')
conn.common.UNI_VERSION
'92'
```

**Exception traceback**:
```
>>> conn.replication.get_storage_group_snapshot_list("SG_OFR730_08_MG_45")
['SG_OFR730_08_MG_45', 'snap840d44db']
>>> conn.replication.get_storage_group_snapshot_list("SG_OFR730_08_MG_60")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/PyU4V/replication.py", line 192, in get_storage_group_snapshot_list
    response = self.get_resource(
  File "/usr/local/lib/python3.8/site-packages/PyU4V/common.py", line 375, in get_resource
    return self.get_request(
  File "/usr/local/lib/python3.8/site-packages/PyU4V/common.py", line 349, in get_request
    self.check_status_code_success(operation, sc, message)
  File "/usr/local/lib/python3.8/site-packages/PyU4V/common.py", line 177, in check_status_code_success
    raise exception.ResourceNotFoundException(
PyU4V.utils.exception.ResourceNotFoundException:
The requested resource was not found: Error GET symmetrix.
The status code received is 404 and the message is {'message': 'No snapshot named found'}.

```
Ideally it should return a empty list as it was in PyU4V 9.1:
```
conn.replication.get_storage_group_snapshot_list("SG_OFR730_08_MG_60")
[]
>>>
>>> conn.common.UNI_VERSION
'91'
>>> conn.common.get_uni_version()
('V9.2.0.6', '92')
```

**Comment by helenwalsh** [comment]
Thanks Gupta, we will will look into this.

**Comment by rawstorage** [comment]
Mukesh, just curious where did you install this version, the latest on pip is 9.2.0.4 and on github it's 9.2.0.5?, just curious

**Comment by mukeshguptakc** [comment]
Hi Paul,

I am not sure, to where the V9.2.0.6 is referring at..

I tried installing PyU4V==9.2.0.4 on a completely new setup.

```
pip install PyU4V==9.2.0.4
Collecting PyU4V==9.2.0.4
  Downloading PyU4V-9.2.0.4-py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 4.2 MB/s
Collecting requests
  Downloading requests-2.25.1-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 9.0 MB/s
Collecting urllib3
  Downloading urllib3-1.26.3-py2.py3-none-any.whl (137 kB)
     |████████████████████████████████| 137 kB 18.0 MB/s
Collecting six
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting prettytable
  Using cached prettytable-2.0.0-py3-none-any.whl (22 kB)
Collecting chardet<5,>=3.0.2
  Downloading chardet-4.0.0-py2.py3-none-any.whl (178 kB)
     |████████████████████████████████| 178 kB 65.7 MB/s
Collecting certifi>=2017.4.17
  Using cached certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
Collecting idna<3,>=2.5
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting wcwidth
  Using cached wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Requirement already satisfied: setuptools in /usr/local/lib/python3.8/site-packages (from prettytable->PyU4V==9.2.0.4) (47.1.0)
Installing collected packages: chardet, certifi, urllib3, idna, requests, six, wcwidth, prettytable, PyU4V
Successfully installed PyU4V-9.2.0.4 certifi-2020.12.5 chardet-4.0.0 idna-2.10 prettytable-2.0.0 requests-2.25.1 six-1.15.0 urllib3-1.26.3 wcwidth-0.2.5
```

And Post that i am still observing the same version as follow:
```
conn
<PyU4V.univmax_conn.U4VConn object at 0x7fe4bbe05790>
>>> conn.common.get_uni_version()
('V9.2.0.6', '92')
```
I have always used `pip install `for getting the` PyU4V Library`

**Comment by mukeshguptakc** [comment]
Hi Paul,

The version 9.2.0.6 is the Unipshere Management console version:
![Screenshot 2021-02-03 at 12 48 27 PM](https://user-images.githubusercontent.com/34023745/106711945-4d7cab00-661e-11eb-8b8e-cda1cf2fba7d.png)

**Comment by rawstorage** [comment]
ok, that makes more sense, it's the unisphere version not PyU4V release.

**Comment by helenwalsh** [comment]
Hi Mukesh,
This has been fixed in PyU4V 9.2.0.7.  Please let us know if you have any further issues.  Thank you.
